lib/acl.c

```diff
@@ -43,17 +43,38 @@ enum {
 	NUM_INPUTS_IPV6,
 };
 
-/* Callback function for when there's no classification match. */
+/* TODO Parse the unmathced packets, and direct them if necessary. */
```

Comment:
It seems like this functionality should be a part of this patch, since we need it for Gatekeeper to be operational in a production deployment.